#slaveNb from 1 to 5
int32 slaveNb
---
trs_msgs/MotorCmdSet motorCmdSet
